web/index.html

```diff
@@ -3,7 +3,8 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="icon" href="favicon-32x32.png" type="image/png" />
+    <link rel="icon" href="src/assets/singlestore-logo-xs.png" type="image/png" />
+    <meta name="twitter:image" content="https://www.figma.com/file/zLZinfGmhUmnA6fWJedQVc/RealTime-Digital-Marketing-App?node-id=887%3A18406&t=x2neHA3Rtfjx7ooc-4">
```

Comment:
Will change this later with non figma link once we have the alternative

Reply:
you might be able to export the image in figma on the right side export option
![image](https://user-images.githubusercontent.com/72758386/214369201-ea98d518-6699-481b-9060-f8d1609a0a31.png)

Reply:
Yes the issue is not with the image

But propogating this image to linkedin post

Its done with this og and twitter tags that I added here
This will only work when the content exist on the link provided. i.e It will be only available when its merged to master